WordPress/gutenberg
# Global Styles: Improve organisation of site background color and image

## What?
Closes: #63096 

## Why?
To improve the organisation for background color and images by creating a new Background Panel in global styles.

## How?
This PR adds a `Background` section which collates both options.

## Testing Instructions
1. Open a Site Editor.
2. Go to global styles.
3. See the Background Panel.

## Screenshots or screencast 
https://github.com/WordPress/gutenberg/assets/77401999/bc5e6c41-1028-4157-a136-4d2b97c5afe1
